### PR TITLE
Nav menu

### PR DESCRIPTION
**Pull request by LikaKoptyreva**
Restructuring and redesigning

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,5 +1,29 @@
 console.clear();
```
Comment (philiprenich) [inline src/scripts/main.js]: This is an odd line to have in a production script.

**Review by philiprenich — CHANGES_REQUESTED** [review]
Will discuss various changes in person

File: src/_layouts/page.html [hunk 2]
```diff
@@ -90,41 +90,29 @@
     <!-- End Google Tag Manager (noscript) -->
 
     <header>
-      <nav class="dark-bg">
-        <input type="checkbox" />
-        <label>&equiv;</label>
-        <a class="home-button" href="/" aria-label="BeakBox Home page">
-          <img class="clickable" alt="Logo" src="/imgs/title.svg" />
+      <nav class="nav">
```
Comment (philiprenich) [inline src/_layouts/page.html]: This class name is redundant to the element. If using the element by itself is too vague, `header > nav` may work instead or a clearer class name.

File: src/scss/_base.scss [hunk 3]
```diff
@@ -19,8 +19,8 @@ body {
   overflow-x: hidden;
 }
 
-img:not(.dark-bg *, .footer *, .blog-posts *),
-svg:not(.dark-bg *, .footer *, .blog-posts *) {
+img:not(.nav *, .footer *, .blog-posts *),
```
Comment (philiprenich) [inline src/scss/_base.scss]: If you change/remove the `<nav>` class name don't forget the corresponding CSS :)

File: src/scss/_header.scss [hunk 4]
```diff
@@ -36,203 +32,148 @@ header {
...
-//* Navigation *//
+  .header-banner {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 5em 2em;
+    width: 100%;
+    width: 80rem;
```
Comment (philiprenich) [inline src/scss/_header.scss:43]: duplicate `width` property.

File: src/scss/_header.scss [hunk 5]
```diff
@@ -36,203 +32,148 @@ header {
...
-    display: none;
-    margin: 0;
-    padding: 0;
-    width: 2rem;
-    height: 2rem;
-    opacity: 0;
-    cursor: pointer;
-  }
-  > label {
+  input[type="checkbox"] {
```
Comment (philiprenich) [inline src/scss/_header.scss:74]: Add a `cursor: pointer` style so sighted users can tell the element is clickable.

File: src/scss/_header.scss [hunk 6]
```diff
@@ -36,203 +32,148 @@ header {
...
-.shop-nav {
-  align-items: center;
-  display: flex;
-}
 
-.shop-nav > svg {
-  height: 1em;
-  margin-right: 1rem;
-}
+    &:checked ~ .nav__links {
```
Comment (philiprenich) [inline src/scss/_header.scss]: Let's go over the nav in person. Easier to talk through a couple things.

File: src/scss/main.scss [hunk 7]
```diff
@@ -52,22 +52,26 @@
 //* Buttons *//
 
 a.button,
-input.button {
-  position: relative;
+input.button,
```
Comment (philiprenich) [inline src/scss/main.scss]: There shouldn't be `input`s with a class of "button". Instead use an actual button or `type="button"` (and selector `input[type="button"]`

File: src/scss/main.scss [hunk 8]
```diff
@@ -52,22 +52,26 @@
...
-  margin: 2em 0;
+  border: 2px solid var(--color-cta-border);
   text-decoration: none;
-  font-weight: inherit;
-  transition: 0.5s ease-in-out;
+  font-weight: var(--font-weight-bold);
+}
+
+a.button:hover,
+input.button:hover,
```
Comment (philiprenich) [inline src/scss/main.scss]: input class here as well

**Review by philiprenich — CHANGES_REQUESTED** [review]
Few changes, overall looks good! Let's have a quick chat to go over a couple other things.

File: src/_layouts/page.html [hunk 9]
```diff
@@ -90,41 +90,29 @@
     <!-- End Google Tag Manager (noscript) -->
 
     <header>
-      <nav class="dark-bg">
-        <input type="checkbox" />
-        <label>&equiv;</label>
-        <a class="home-button" href="/" aria-label="BeakBox Home page">
-          <img class="clickable" alt="Logo" src="/imgs/title.svg" />
+      <nav class="main-nav">
```
Comment (LikaKoptyreva) [inline src/_layouts/page.html:93]: I've decided to keep the class here but renamed it to "main-nav" - to keep using BEM and differentiate it from the nav element in the blog section (I might see if we can improve it in the next pr)

File: src/scss/_base.scss [hunk 10]
```diff
@@ -73,7 +66,48 @@ li + li {
   color: var(--color-red);
 }
 
-*:focus {
-  outline: 3px dashed #000;
-  outline-offset: 2px;
+//* Focus set up *//
+
+:is(a, button, input, textarea, summary) {
```
Comment (LikaKoptyreva) [inline src/scss/_base.scss:71]: Checked browser compatibility of :is and it shouldn't cause any issues - if someone uses an older version of the browser that doesn't support :is, it shouldn't be an issue too (as far as I understand) as focus styles will fall back to the browser's default

File: src/scss/_footer.scss [hunk 11]
```diff
@@ -14,10 +14,7 @@
     grid-template-areas:
       "logo links1 links2 copyright"
       "socials links1 links2 copyright";
-  }
-
-  a:focus {
-    outline-color: white;
+    justify-items: center;
```
Comment (LikaKoptyreva) [inline src/scss/_footer.scss:17]: Added this property to make the link area of the logo the same size as the logo

File: src/scss/_footer.scss [hunk 12]
```diff
@@ -60,6 +57,11 @@
       grid-area: socials;
     }
 
+    a {
+      display: inline-block;
+      height: 4rem;
+    }
+
```
Comment (LikaKoptyreva) [inline src/scss/_footer.scss:64]: Added these properties to make the social media links' area the same size as the image

**Review by philiprenich — APPROVED** [review]

**Pull request merged**